client/login/magic-login/emailed-login-link-successfully.jsx

```diff
@@ -63,24 +57,34 @@ class EmailedLoginLinkSuccessfully extends Component {
 					waitForEmailAddress={ emailAddress }
 				/>
 
-				<h1 className="magic-login__form-header">{ translate( 'Check your email!' ) }</h1>
+				<h1 className="magic-login__form-header">{ translate( 'Check your email' ) }</h1>
 
 				<Card className="magic-login__form">
-					<p>{ preventWidows( line ) }</p>
+					<div className="magic-login__form-text">
+						<p>{ preventWidows( line ) }</p>
+						<b>{ emailAddress }</b>
+					</div>
 				</Card>
+				<div className="magic-login__emails-list">
+					<MagicLoginEmailWrapper emailAddress={ emailAddress } />
+				</div>
 
 				<div className="magic-login__footer">
-					<a
-						href={ login( {
-							isJetpack: this.props.isJetpackLogin,
-							isWhiteLogin: this.props.isWhiteLogin,
-							locale: this.props.locale,
-						} ) }
-						onClick={ this.onClickBackLink }
-					>
-						<Gridicon icon="arrow-left" size={ 18 } />
-						{ translate( 'Back to login' ) }
-					</a>
+					<p>
+						{ translate(
+							" Didn't get the email? You might want to double check if the email address is associated with your account,"
```

Comment:
```suggestion
							"Didn't get the email? You might want to double check if the email address is associated with your account,"
```

Is there is a reason for the `,` in the end?

Reply:
Yes, this is just about to make a short stop in the sentence. I have removed it from the translation and made it part of the `p` as a string
Design reference :
<img width="362" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/345b490c-cb54-498d-ab35-4b6ae37c88aa">

Reply:
Same issue. We shouldn't concat UI strings.

Reply:
Fixed.